errors/minification-disabled.md

```diff
@@ -0,0 +1,18 @@
+# Minification disabled in production
+
+#### Why This Error Occurred
+
+When overriding `optimization.minimize` or `optimization.minimizer` incorrectly in `next.config.js`, this degrades performance of your application
+
+#### Possible Ways to Fix It
+
+This is not a bug in Next.js, it's related to the user adding a custom webpack config to `next.config.js` and modyfing `optimization.minimize` or `optimization.minimizer`. Solution would be:
+
+```js
```

Comment:
Drop this example, it's not representative of when this would happen IMO. I want to make sure we see what users are doing when they change this value.